ui/README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
```

Comment:
Not sure we need to commit this?

This looks like standard copypasta.

We can only add the parts we need to the root readme.

Reply:
this is auto genated